src/Compilers/Core/Portable/Compilation/CompilationOptions.cs

```diff
@@ -166,7 +166,7 @@ public abstract class CompilationOptions
         /// <summary>
         /// Emit mode that favors debuggability. 
         /// </summary>
-        internal bool DebugPlusMode { get; private protected set; }
+        internal bool DebugPlusMode { get; set; }
```

Comment:
We should mark #44809 fixed as a part of this change.

Reply:
Done